### ISSUE TITLE
ARIA: tab role — guidance about aria-expanded is wrong

### ISSUE THREAD

**Issue by tonyhallett** [mdn/content#15516]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

### What specific section or headline is this issue about?

Description

### What information was incorrect, unhelpful, or incomplete?

> When a tab is selected or active, its corresponding controlled tabpanel should have its [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) attribute set to true and its hidden attribute set to false, otherwise the reverse.

### What did you expect to see?

aria-expanded is for multi-selectable

### Do you have any supporting links, references, or citations?

https://w3c.github.io/aria/#tab

> For a multi-selectable [tablist](https://w3c.github.io/aria/#tablist), authors SHOULD ensure that the [tab](https://w3c.github.io/aria/#tab) for each visible tabpanel has the aria-expanded attribute set to true, and that the tabs associated with the remaining hidden [tabpanel](https://w3c.github.io/aria/#tabpanel) elements have their [aria-expanded](https://w3c.github.io/aria/#aria-expanded) [attribute](https://w3c.github.io/aria/#dfn-attribute)s set to false.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tab_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tab_role/index.md
* Last commit: https://github.com/mdn/content/commit/ea032ab86280848d3842a7e4455b831ea2545603
* Document last modified: 2022-03-15T02:11:20.000Z

</details>
